hvac/api/auth_methods/oidc.py

```diff
@@ -45,6 +45,7 @@ def create_role(
         token_period=None,
         token_type=None,
         path=None,
+        user_claim_json_pointer=False,
```

Comment:
```suggestion
        user_claim_json_pointer=None,
```
if we are changing the JWT method to `None` then I suppose we should change this one too.